ts/input/tex/cases/CasesConfiguration.ts

```diff
@@ -139,32 +175,38 @@ export const NumcasesMethods = {
     //
     //  Process the second column as text and continue parsing from there,
     //
-    const text = tex.substr(parser.i, i - parser.i);
+    const text = tex.substr(parser.i, i - parser.i).replace(/^\s*/, '');
     parser.PushAll(ParseUtil.internalMath(parser, text, 0));
     parser.i = i;
   }
 
 };
 
-new EnvironmentMap('numcases-env', EmpheqUtil.environment, {
+/**
+ * The environments for this package
+ */
+new EnvironmentMap('cases-env', EmpheqUtil.environment, {
   numcases: ['NumCases', 'cases'],
   subnumcases: ['NumCases', 'cases']
 }, NumcasesMethods);
 
-new MacroMap('numcases-macros', {
+/**
+ * The macros for this package
+ */
+new MacroMap('cases-macros', {
   '&': 'Entry'
 }, NumcasesMethods);
 
 //
 //  Define the package for our new environment
 //
-export const NumcasesConfiguration = Configuration.create('numcases', {
+export const NumcasesConfiguration = Configuration.create('cases', {
```

Comment:
The export is still called NumcasesConfiguration which seems odd.

Reply:
Thanks.  I missed that one, and will make a new commit to fix it.